zest/releaser/utils.py

```diff
@@ -695,9 +695,17 @@ def _execute_command(command, input_value=''):
     stderr_output = e.read()
     # We assume that the output from commands we're running is text.
     if not isinstance(stdout_output, six.text_type):
-        stdout_output = stdout_output.decode(OUTPUT_ENCODING)
+        try:
+            stdout_output = stdout_output.decode(OUTPUT_ENCODING)
+        except UnicodeDecodeError:
+            stdout_output = '%s stdout output is not a valid %s stream.' % (
```

Comment:
So... instead of getting back output from, for instance, "git diff" we now get an error message?

This silences the error: is this **always** a good thing?